### ISSUE TITLE
Demon's Souls[NPEB01202] Run psn versions as disc and decrypt extension-less sdat files fix request

### ISSUE THREAD

**Issue by LittleStinky123** [RPCS3/rpcs3#6342]
I noticed this on rpcs3 0.0.6-8465-d48d424b.
All files in the USRDIR directory of the PSN release of Demon's Souls are encrypted as finalized v3 sdats while other releases only have specific files encrypted and have an extension to signal this.
These files with Demon's Souls don't have any extension of this kind.
When I create the directory structure of a disc, rpcs3 boots the game, but doesn't decrypt the extension-less sdat files so it doesn't continue after the initial loading screen.
Decrypting those sdat files with 
```
$oldfiles = Get-ChildItem "ABSOLUTE_PATH_TO_USRDIR\*.*" -Recurse -exclude EBOOT.BIN,*.sdat,*.*_tmp;
foreach ($oldfile in $oldfiles) {
    $prefix = Split-Path -Path $oldfile;
	$filetype = [System.IO.Path]::GetExtension($oldfile);
	$length = $filetype.length;
	$filetype2 = -join($filetype, "_tmp");
	$newfilesuffix = -join($oldfile.BaseName, $filetype2);
    $newfile = Join-Path -Path $prefix -ChildPath $newfilesuffix;
	.\make_npdata.exe -v -d $oldfile $newfile 0;
	if([System.IO.File]::Exists($newfile)){
		Move-Item -LiteralPath $newfile -Destination $oldfile -Force;
	}
}
```
makes the game run flawlessly as a disc inside rpcs3, the elf of the disc and psn release have the same md5sum.

The fps patch also works because of this.
I can confirm that this works for all regional versions of Demon's Souls that are on PSN.

I use an 8700k with a 1060 6gb on windows 10 64bit.

**Comment by alessiot89** [comment]
This is also extremely slow since it will open and close make_npdata for each file in USRDIR. NPEB01202 is about more then 12K files.

**Comment by LittleStinky123** [comment]
Well creating a disc structure includes creating a game directory with a name of your liking, creating the PS3_GAME directory in it and moving the decrypted files from NPEB01202 for example to PS3_GAME and putting a generic PS3_DISC.SFB file on the same level as PS3_GAME and pasting the id you got from the param.sfo to hex address 0x220 of PS3_DISC.SFB.

**Comment by elad335** [comment]
Fixed by #10849
